desktop/menus/view-menu.js

```diff
@@ -98,6 +98,10 @@ const buildViewMenu = settings => {
       {
         label: 'T&heme',
         submenu: [
+          {
+            label: '&System',
```

Comment:
Is this the label we use on the other apps? Is `System` enough to explain `System setting`?

Reply:
I struggled with what to call this option for the setting. Is it `Default`? Is it `OS Setting`? System seemed to be the best answer.